### PR TITLE
Mabdullahlkhalil/gemdependacies

### PR DESCRIPTION
**Pull request by mabdullahkhalil**

### CODE REVIEW

**Comment by q9f** [comment]
Thank you. I'm fixing the money tree gem right away. https://github.com/GemHQ/money-tree/pull/57

Can you rebase this on `q9f/eth/openssl` branch?

**Comment by mabdullahkhalil** [comment]
> Thank you. I'm fixing the money tree gem right away. [GemHQ/money-tree#57](https://github.com/GemHQ/money-tree/pull/57)
> 
> Can you rebase this on `q9f/eth/openssl` branch?

@q9f I did, can you check if it is done

File: eth.gemspec [hunk 1]
```diff
@@ -30,15 +30,15 @@ Gem::Specification.new do |spec|
...
-  spec.add_dependency 'money-tree', '~> 0.10'
-  spec.add_dependency 'openssl', '~> 2.2'
+  spec.add_dependency 'money-tree', '~> 0.11.0'
+  spec.add_dependency 'openssl', '~> 3.0'
   spec.add_dependency 'rlp', '~> 0.7'
   spec.add_dependency 'scrypt', '~> 3.0'
 
   spec.platform = Gem::Platform::RUBY
-  spec.required_ruby_version = ">= 2.2", "< 4.0"
+  spec.required_ruby_version = ">= 3.0", "< 4.0"
```
Comment (q9f) [inline eth.gemspec]:
```suggestion
  spec.required_ruby_version = ">= 2.6", "< 4.0"
```

File: eth.gemspec [hunk 2]
```diff
@@ -30,15 +30,15 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
   spec.test_files    = spec.files.grep %r{^(test|spec|features)/}
 
-  spec.add_dependency 'keccak', '~> 1.2'
+  spec.add_dependency 'keccak', '~> 1.3'
   spec.add_dependency 'ffi', '~> 1.15'
-  spec.add_dependency 'money-tree', '~> 0.10'
-  spec.add_dependency 'openssl', '~> 2.2'
+  spec.add_dependency 'money-tree', '~> 0.11.2'
```
Comment (q9f) [inline eth.gemspec]:
```suggestion
  spec.add_dependency 'money-tree', '~> 0.11'
```

**Comment by q9f** [comment]
Thank you! Let me try to release this.

**Pull request merged**